docs/modules/ROOT/pages/index.adoc

```diff
@@ -788,131 +788,80 @@ Such updates to the specification SHOULD continue to use the `cawg.identity` ass
-
-[#verifiable-presentation-example]
-.Verifiable presentation example
-[example]
-====
-In this example, an _<<_actor,actor>>_ who we will call Alice:
-
-* Is using an authoring tool provided by an _<<_identity_claims_aggregator,identity claims aggregator>>,_ and
-* Wishes to use a credential held in a digital wallet that she controls to document her role in creating one or more _<<_c2pa_asset,C2PA assets>>_ with that authoring tool.
-
-IMPORTANT: This example uses a W3C Verifiable Presentation, but the workflow is similar for other forms of authentication that may be used (OAuth2 or OpenID Connect, for example).
-
-[#ica-workflow-demonstrate-identity]
-*Workflow 1: Alice demonstrates her identity to the identity claims aggregator*
-
-Alice must first prove her identity to the _<<_identity_claims_aggregator,identity claims aggregator.>>_ This must be done prior to the creation of any *<<_identity_assertion,identity assertions>>.*
-
-[mermaid,width=100%]
-....
-sequenceDiagram
-  autonumber
-  participant A as Alice
-  participant W as Wallet
-  participant ICA as Identity claims aggregator
-
-  A ->> ICA: Request connection to ICA
-  ICA ->> A: Presentation request link
-
-  A ->> W: Follow this link
-  W ->> ICA: Follow link
-  ICA ->> W: Presentation request
-  W ->> A: Consent to share info?
-  A ->> W: Approve
-  W ->> ICA: Verifiable presentation
-  ICA ->> ICA: Verifies and remembers presentation
-....
+_This section is non-normative._
 
-The interactions involving Alice are presumed to be visible user experience interactions that she encounters. The interactions between wallet and _<<_identity_claims_aggregator,identity claims aggregator>>_ are presumed to be network traffic.
+Content creators (_<<_named_actor,named actors>>_) may wish to document their role in creating a _<<C2PA asset>>_ using identity signals that are commonly understood in the mass market. Examples of such signals include:
 
-1. Alice initiates a connection request via an interface provided by the _<<_identity_claims_aggregator,identity claims aggregator>>._
+* verified web sites
+* social media accounts
+* official ID documentation
+* professional accreditations
+* organizational affiliations
 
-2. The _<<_identity_claims_aggregator,identity claims aggregator>>_ responds with a link (possibly a QR code) which points to a link:https://w3c-ccg.github.io/vp-request-spec/[Verifiable Presentation Request].
+These common identity signals, though popular, are not well-designed for use as lasting identifiers. Some of the challenges associated with these identity signals include:
 
-3. Alice then instructs her digital wallet to follow the QR code or link.
+* The methods for accessing, describing, and presenting these signals are widely disparate.
```

Comment:
affiliations not signals

Reply:
I like "signal"; the presented information might not be an affiliation but a self-asserted statement (e.g., owning my own website)